Bottom: 15bc4ccc76f3f588234e9fdf4a35288722134f6d
Top:    9e47abd9e6c71f6236759712df0f1d5d4485659b
Author: scott snyder <sss@karma>
Date:   2020-02-22 23:57:51 -0500

Refresh of gws

---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GirlswithSlingshots.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GirlswithSlingshots.java
index b94476e..bdd9044 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GirlswithSlingshots.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/GirlswithSlingshots.java
@@ -31,11 +31,11 @@ public class GirlswithSlingshots extends IndexedComic {
 
 	@Override
 	protected String getFrontPageUrl() {
-		return "http://www.girlswithslingshots.com/";
+		return "https://www.girlswithslingshots.com/";
 	}
 
 	public String getComicWebPageUrl() {
-		return "http://www.girlswithslingshots.com/";
+		return "https://www.girlswithslingshots.com/";
 	}
 
 	@Override
@@ -64,7 +64,9 @@ public class GirlswithSlingshots extends IndexedComic {
 	@Override
 	public int addException(int id, int increment) {
 		while(true) {
-			if(!mHash.containsKey(id)) {
+			if(!mHash.containsKey(id) &&
+                           (id <= 1293 || id >= 1924))
+                        {
 				break;
 			}
 			id += increment;
@@ -75,7 +77,7 @@ public class GirlswithSlingshots extends IndexedComic {
 	@Override
 	public String getStripUrlFromId(int num) {
 //Log.d("GWS", "num " + num);
-		return "http://www.girlswithslingshots.com/comic/gws-chaser-" + num;
+		return "https://www.girlswithslingshots.com/comic/gws-chaser-" + num;
 	}
 
 	@Override
